variants/STM32H5xx/H503RBT/variant_NUCLEO_H503RB.h

```diff
@@ -157,13 +156,14 @@
   #define PIN_SERIAL_TX         PA4
 #endif
 
-#define HSE_VALUE               24000000UL /*!< Value of the External oscillator in Hz */
-
 // Extra HAL modules
 #if !defined(HAL_DAC_MODULE_DISABLED)
   #define HAL_DAC_MODULE_ENABLED
 #endif
 
+// Value of the HSE Bypass in Hz
+#define HSE_VALUE               24000000UL
```

Comment:
This is not inline with your change using HSE Bypass as it is 8 MHz.
Anyway, as you have to revert to CSI then it will be ok.